install_config/monitoring/modules/enabling_dynamically-provisioned_storage.adoc

```diff
@@ -0,0 +1,22 @@
+= Enabling dynamically-provisioned storage
+
+Instead of statically-provisioned storage, you can use dynamically-provisioned storage.
+
+.Procedure
+
+To enable dynamic storage for Prometheus and Alertmanager, set the following parameters to `true` in the Ansible inventory file:
```

Comment:
There's no Ansible inventory file anymore. :)

Please make these explicit steps.

Reply:
No, there isn't :) Just tried to keep this PR about restructuring only.